src/entities/communitySbt.ts

```diff
@@ -25,10 +30,37 @@ type MultiRedeemData = {
     roots: Array<Bytes>;
 }
 
+enum TxBadgeStatus {
+    SUCCESSFUL,
+    FAILED,
+    PENDING
+}
+
+enum BadgeClaimingStatus {
```

Comment:
can you please export this type?

Reply:
exported 4e3322a4407590640eef146a8b7f7999f83249eb